layouts/partials/sections/contact.html

```diff
@@ -22,12 +22,13 @@ <h1><span class="icon icon-mail"></span>{{ i18n "letsChat" }}</h1>
             </select>
       </div>
       {{- end }}
-      <div class="col-xs-12{{ if .Page.Params.selectKeys }} col-sm-6{{ end }}"><input type="text" name="_subject" aria-label="email subject" placeholder="{{ i18n "subject" }}" required></div>
+      <div class="col-xs-12{{ if .Page.Params.selectKeys }} col-sm-6{{ end }}"><input type="text" name="subject" aria-label="email subject" placeholder="{{ i18n "subject" }}" required></div>
+      <!-- Note: in Chrome ... "An invalid form control with name='message' is not focusable." -->
       <div class="col-xs-12">
-          <textarea name="message" aria-label="Please enable JS" required></textarea>
+          <textarea name="message" aria-label="Please enable JS" ></textarea>
```

Comment:
This is part of the [honeypot](https://en.wikipedia.org/wiki/Honeypot_(computing)) that is meant to attract spammers, as described under "Spambot protection" [in the README](https://github.com/kdevo/osprey-delight#spambot-protection). I think you've also fallen into the honeypot here by thinking it's the actual message field. 
That's my mistake, as I did not leave a comment here to highlight that this is handled via JS instead by manipulating the DOM so that `message2` is the actual message field: 

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/js/src/contact.js#L2-L12

Sorry for that, I'll leave a comment above to clarify it.